core/modules/cart/store/actions.ts

```diff
@@ -751,6 +769,15 @@ const actions: ActionTree<CartState, RootState> = {
   },
   toggleMicrocart ({ commit }) {
     commit(types.CART_TOGGLE_MICROCART)
+  },
```

Comment:
These methods are related to UI and therefore should be in the theme / Vuex store in the theme not in the core. Please move them there please